### PR TITLE
feat: Add OpenCode plugin for AxonHub tracing with configurable headers. --issue=#1

### LINKED ISSUE #1: FEATURE: perf the tracing header of opencode.
Related: https://github.com/looplj/axonhub/issues/837#issuecomment-3906792685

### PR DESCRIPTION
**Pull request by zouxingyuks**
- Introduced `OpenCodeAxonHubTracing` plugin to inject tracing headers into LLM requests.
- Configurable header keys via environment variables.
- Added tests for header injection functionality.
- Updated package dependencies and configurations.
- Created documentation for usage and installation.

closed #1

## Summary by CodeRabbit

* **New Features**
  * Environment variable support for customizable trace header names with sensible defaults
  * Improved robustness for missing/partial input to header injection

* **Documentation**
  * Added comprehensive plugin docs and localized README to Chinese with updated configuration examples

* **Tests**
  * Added comprehensive test suite covering header injection scenarios and edge cases

* **Chores**
  * Updated package metadata and repository info, bumped version to 0.1.4
  * Added TypeScript build configuration and build/script updates

### CODE REVIEW

**Review by looplj — APPROVED** [review]

File: index.test.ts [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+    const output = { headers: {} as Record<string, string> }
+
+    await hook(
+      { sessionID: "ses_blank", message: { id: "msg_blank" } } as any,
+      output,
+    )
+
+    expect(output.headers["AH-Thread-Id"]).toBe("ses_blank")
+    expect(output.headers["AH-Trace-Id"]).toBe("msg_blank")
+  })
```
[the remark this answers is not in the document]
Comment (zouxingyuks) [reply index.test.ts:129]: this is test.
